target/linux/rockchip/modules.mk

```diff
@@ -0,0 +1,8 @@
+#
+# Copyright (C) 2019 OpenWrt.org
```

Comment:
I'm not really a fan of copying those (outdated) copyright lines over from source files, particularly when it's trivial files.